Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM sub2u/ruby-2.3.1-node-6.8.0
+MAINTAINER sub2u
+
+COPY ./ app/
```

Comment:
I was meaning not `ADD`, but [volumes](https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-host-directory-as-a-data-volume). Copying process takes time and in development copy files on change is kinda pricy. If volume is used, then it reads the files directly from the mounted volume, so it should be faster.